modules/i3s/src/i3s-loader.js

```diff
@@ -53,20 +53,20 @@ const I3SLoader = {
 };
 
 async function parse(data, options, context, loader) {
-  const url = new URL(context.url);
+  const url = context.url;
   options.i3s = options.i3s || {};
 
   // auto detect file type based on url
   let isTileset;
   if (options.i3s.isTileset === 'auto') {
-    isTileset = TILESET_REGEX.test(url.pathname);
+    isTileset = TILESET_REGEX.test(url);
```

Comment:
Should this be TILESET_LAYER_REGEX?

Reply:
"Layer" in I3S is the corresponding entity for "tileset" in 3DTiles. So 'TILESET_LAYER_REGEX' means 'TILESET_TILESET_REGEX'. I think 'layer' was called 'tileset' in I3SLoader to match entities with Tiles3DLoader. It makes sense in my opinion.